components/PaymentOutcome/index.tsx

```diff
@@ -48,6 +67,38 @@ function PaymentOutcome({ paymentRequest, paymentAmount, dispatch }: Props) {
     </button>
   )
 
+  const downloadReceipt = (
+    <button className={styles.pay_new_btn} onClick={() => printReceipt()}>
```

Comment:
Can't we just pass the function directly to onClick?
```suggestion
    <button className={styles.pay_new_btn} onClick={printReceipt}>
```